Sources/ArcGISToolkit/Components/Authentication/CertificatePickerViewModifier.swift

```diff
@@ -212,23 +253,11 @@ private extension View {
                 viewModel.cancel()
             }
         } message: {
-            Text(message(for: viewModel.certificateImportError))
-        }
-    }
-    
-    func message(for error: CertificateImportError?) -> String {
-        let defaultMessage = "The certificate file or password was invalid."
-        guard let error = error else {
-            return defaultMessage
-        }
-        
-        switch error {
-        case .invalidData:
-            return "The certificate file was invalid."
-        case .invalidPassword:
-            return "The password was invalid."
-        default:
-            return defaultMessage
+            Text(viewModel.certificateError?.localizedDescription ?? String(
+                    localized: "The certificate file or password was invalid.",
+                    bundle: .module
+                 )
```

Comment:
Even though it isn't, the indentation really looks off here. A little refactoring would address that.

```suggestion
            Text(
                viewModel.certificateError?.localizedDescription ?? String(
                    localized: "The certificate file or password was invalid.",
                    bundle: .module
                )
```

Reply:
done